java-connectors/kafka-connect-common/src/main/java/io/lenses/streamreactor/common/config/base/ConfigSettings.java

```diff
@@ -0,0 +1,28 @@
+package io.lenses.streamreactor.common.config.base;
```

Comment:
unfortunately we'll have to add header here by `gradlew updateLicenses` (we'll make it automatic soon)